### PR TITLE
Drop PHP 8.0, require PHP 8.1 minimum, use PHPUnit 10

### PR DESCRIPTION
**Pull request by ruudk**

### CODE REVIEW

**Comment by ruudk** [comment]
stupid bot

**Comment by ruudk** [comment]
@mcg-web ready for review

File: phpunit.xml.dist.bak [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mcg-web) [inline phpunit.xml.dist.bak:1]: can we delete this one ?

Comment (ruudk) [reply phpunit.xml.dist.bak:1]: Sorry, will do

Comment (ruudk) [reply phpunit.xml.dist.bak]: done

**Pull request merged**